docs/content/users/debugging-profiling/step-debugging.md

```diff
@@ -66,6 +66,9 @@ Click on the local repository path and add `/var/www/html` as the *Absolute path
 6. Start debugging by clicking the “debug” button, which will launch a page in your browser:
     ![PhpStorm debug start](../../images/phpstorm-config-debug-button.png)
 
+!!!tip "If you’re using [additional_hostnames](../usage/extend/additional-hostnames.md)"
+    If you also want xdebug to run on additional hostnames. Simply create a copy of the server with the current settings and change the host and name to the desired ones. This will tell PHP Storm to listen to that hostname with the appropriate mapping.
```

Comment:
```suggestion
!!!tip "If you’re using [additional_hostnames](../usage/extend/additional-hostnames.md)"
    If you also want xdebug to trigger when using a URL other than the primary URL (as with  additional hostnames), create a copy of the PhpStorm `server` (Settings->PHP->Servers) with the current settings and change the host and name to the desired ones. This will tell PHP Storm to listen to that hostname with the appropriate mapping.
```